wordpress-mobile/WordPress-iOS
# Aztec: Placeholder

### Details:
In this PR we're implementing a simple Placeholder, to be displayed behind the Aztec Editor. Plus, i've done a small cleanup here and there (grouped UIColor's / UIFont's together!)

Fixes #6634

### To test:
1. Launch Aztec
2. Verify that the Placeholder shows up onscreen
3. Type some text, and verify that the placeholder disappears
4. Empty the Editor's contents
5. Switch to HTML mode
6. Verify that the placeholder gets hidden as well

Note: Step 6 is consistent with the hybrid, but i'd be more than happy to fine tune, as needed!

Needs review: @astralbodies 
Thanks in advance Aaron!!
